src/CacheCleaner/AbstractRequestBuilder.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Spatie\ResponseCache\CacheCleaner;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+
+abstract class AbstractRequestBuilder
+{
+    protected string $method = 'GET';
+    protected array $parameters = [];
+    protected array $cookies = [];
+    protected array $server = [];
+    protected ?string $cacheNameSuffix = null;
+
+    /**
+     * Set the value of method
+     *
```

Comment:
This comment doesn't add any value, remove it.

Reply:
Remove similar further comments as well.

Reply:
ok